erpnext/crm/doctype/timezone/timezone.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Timezone is quite generic and need not be a part of CRM exclusively. We can move this doctype to Frappe.